sandbox/pages/StatusChartPanelPage.qml

```diff
@@ -18,11 +18,11 @@ Item {
         property real minStep: 12000
         property real maxStep: 22000
 
-        property var graphTabsModel: [{text: "Price", enabled: true}, {text: "Balance", enabled: false}]
-        property var timeRangeTabsModel: [{text: "1H", enabled: true},
-                {text: "1D", enabled: true},{text: "7D", enabled: true},
-                {text: "1M", enabled: true}, {text: "6M", enabled: true},
-                {text: "1Y", enabled: true}, {text: "ALL", enabled: true}]
+        property var graphTabsModel: [{text: qsTr("Price"), enabled: true}, {text: qsTr("Balance"), enabled: false}]
+        property var timeRangeTabsModel: [{text: qsTr("1H"), enabled: true},
+                {text: qsTr("1D"), enabled: true},{text: qsTr("7D"), enabled: true},
+                {text: qsTr("1M"), enabled: true}, {text: qsTr("6M"), enabled: true},
+                {text: qsTr("1Y"), enabled: true}, {text: qsTr("ALL"), enabled: true}]
```

Comment:
No, you didn't understand; no need for translating the sandbox app; just do this in status-desktop :)

Reply:
I did understand but overthought of "source" :) so yeah no needed here, removed!